tests/scripts/vg_wrap/valgrind

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Wrap valgrind to run tests from a temp directory. This allows tests to be run in parallel without
+# corrupting each other's temp files.
+
```

Comment:
Many test suites need to read files in the `data_files` subdirectory.

Reply:
As above, it will pick this up from `tests`